### PR TITLE
Add GetSessionId API

### PR DESCRIPTION
**Pull request by vivanov**
An attempt to implement following issue: https://github.com/DCS-gRPC/rust-server/issues/143

### CODE REVIEW

File: src/rpc/mission.rs [hunk 1]
```diff
@@ -165,6 +166,35 @@ impl MissionService for MissionRpc {
...
+        let response = self
+            .get_user_flag(Request::new(trigger::v0::GetUserFlagRequest {
+                flag: String::from("dcs_grpc_session_id"),
+            }))
+            .await?
+            .into_inner();
+        let current_session_id = response.value as i64;
+        if current_session_id == 0 {
+            let ts = OffsetDateTime::now_utc().unix_timestamp();
+            let _result = self
```
Comment (rkusa) [inline src/rpc/mission.rs:183]: I think the following can be removed
```suggestion
            self
```

File: protos/dcs/mission/v0/mission.proto [hunk 2]
```diff
@@ -76,6 +76,10 @@ service MissionService {
   // See https://wiki.hoggitworld.com/view/DCS_func_removeItemForGroup
   rpc RemoveGroupCommandItem(RemoveGroupCommandItemRequest)
       returns (RemoveGroupCommandItemResponse) {}
+
+  // Returns value of the dcs_grpc_session_id flag or current unixtime if it's not set
```
Comment (rkusa) [inline protos/dcs/mission/v0/mission.proto:80]: Suggestion (basically fewer implementation details):
```suggestion
  // Returns an ID for the current session. The ID will change upon mission change or server restart.
```

**Review by rkusa — COMMENTED** [review]
Thanks for the PR, just two minor suggestions 👍  (and protolint wants something: https://github.com/DCS-gRPC/rust-server/runs/6860585828?check_suite_focus=true)

File: src/rpc/mission.rs (hunk 1 above)
Comment (vivanov) [reply src/rpc/mission.rs]: Good point, updated

File: protos/dcs/mission/v0/mission.proto (hunk 2 above)
Comment (vivanov) [reply protos/dcs/mission/v0/mission.proto]: Good point, updated

**Review by rurounijones — CHANGES_REQUESTED** [review]
@vivanov Thanks for the contribution and sorry for the delay in testing. As far as I can tell everything is hunky-dory so thank you very much.

Could you update the CHANGELOG.md to include your change and then I will merge

**Review by rurounijones — APPROVED** [review]

**Pull request merged**